packages/@aws-cdk/cloudformation-include/lib/cfn-include.ts

```diff
@@ -13,6 +13,37 @@ export interface CfnIncludeProps {
    * Both JSON and YAML template formats are supported.
    */
   readonly templateFile: string;
+
+  /**
+   * Specifies the template files that define nested stacks that should be included.
+   *
+   * If your template specifies a stack that isn't included here, it won't be created as a NestedStack
+   * resource, and it won't be accessible from getNestedStack().
+   *
+   * If you include a stack here with an ID that isn't in the template,
+   * template creation will fail and an error will be thrown.
+   *
+   * If you include an ID here that exists in the template but isn't in the template,
```

Comment:
"If you include an ID here that exists in the template but isn't in the template" - quite Zen 😃
(I assume it's a typo)